src/entryPoints/AcceptInvite.res

```diff
@@ -75,11 +74,7 @@ let make = () => {
         ]->getJsonFromArrayOfJson
       let res = await updateDetails(url, body, Post, ())
       let email = HSLocalStorage.getFromMerchantDetails("email")
-      let token = HyperSwitchAuthUtils.parseResponseJson(
-        ~json=res,
-        ~email,
-        ~isAcceptInvite=acceptInvite,
-      )
+      let token = HyperSwitchAuthUtils.parseResponseJson(~json=res, ~email, ~isAcceptInvite=true)
```

Comment:
`~isAcceptInvite=true` This is true in all the places so do we need this param ?